eungabi/src/commonMain/kotlin/com/easternkite/eungabi/navigation/NavOptions.kt

```diff
@@ -15,18 +15,58 @@
  */
 package com.easternkite.eungabi.navigation
 
+/**
+ * The EunGabi Navigation Options.
+ * It holds the route to be popped up.
+ */
 data class NavOptions(
+    /**
+     * The backing property for [popUpToRoute].
+     *
+     * The route to be popped up.
+     * In the [EunGabiController.navigateUp], this is used to get previous entity.
+     */
     private var _popUpToRoute: String = "",
+
```

Comment:
🚫 **[ktlint]** <standard:no-blank-line-in-list> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected blank line(s) in value parameter list
<!-- __reviewdog__:ChA3YjMwNTVmZGE0MWU3MzlhEgZrdGxpbnQ= -->